### PR TITLE
Brian pages structure

### PR DESCRIPTION
**Pull request by pasadyaguy**
Created basic structure for pages.

### CODE REVIEW

File: angular-material-starter/.vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "editor.defaultFormatter": "esbenp.prettier-vscode",
+    "editor.codeActionsOnSave": {
+        "source.organizeImports": true
+    }
+}
```
Comment (MichaelAThew) [inline angular-material-starter/.vscode/settings.json:6]: This looks pretty ok

Comment (pasadyaguy) [reply angular-material-starter/.vscode/settings.json:6]: Why thank you good sir

File: angular-material-starter/angular.json [hunk 2]
```diff
@@ -20,15 +20,10 @@
             "outputPath": "dist/angular-material-starter",
             "index": "src/index.html",
```
Comment (MichaelAThew) [inline angular-material-starter/angular.json:21]: This is a review

**Review by MichaelAThew — APPROVED** [review]

**Review by pasadyaguy — COMMENTED** [review]
Test

**Review by MichaelAThew — APPROVED** [review]

**Pull request merged**